lambda/templates/export_authoriser_policy.json.tpl

```diff
@@ -19,6 +19,16 @@
         "kms:Decrypt"
       ],
       "Resource": "${kms_arn}"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateNetworkInterface",
+        "ec2:DescribeNetworkInterfaces",
+        "ec2:DeleteNetworkInterface",
+        "elasticfilesystem:ClientWrite"
```

Comment:
What does the export authoriser do with EFS?

Reply:
Nothing at all :smile: I'll remove that permission.